src/analytics/ValoraAnalytics.ts

```diff
@@ -90,6 +98,22 @@ class ValoraAnalytics {
     } catch (error) {
       Logger.error(TAG, `Segment setup error: ${error.message}\n`, error)
     }
+
+    try {
+      const { accountAddress } = getCurrentUserTraits(store.getState())
+      const stasigUser = accountAddress
+        ? {
+            userID: accountAddress,
+          }
+        : null
+
+      await Statsig.initialize(STATSIG_API_KEY, stasigUser, {
+        overrideStableID: uniqueID,
```

Comment:
just for my understanding, is this the ID that is used for bucketing for this experiment since we won't have a wallet address? And can we specify what ID to use for bucketing on the experiment configuration?

Reply:
Yes that's the ID that is used for the experiment. Also, yes you can specify whatever ID you'd like to use in the Statsig web app.